spec/lib/slackify_spec.rb

```diff
@@ -41,12 +41,18 @@ module Slackify
 
       let(:text) { context.fetch(:slack_text) }
 
-      let(:builded_payload) {
-        Payload.build(context, :success)
+      def buildedPayload(channel)
```

Comment:
Can we please revert the test changes here and instead just add the `channel` variable to `Payload.build` to match the interface changes?   We don't need to iterate over `Array(context.fetch(:slack_channel))` here in the test, as we're only passing a single channel to `Payload` for each iteration.